components/support/nimbus-fml/src/backends/swift/templates/FeatureManifestTemplate.swift

```diff
@@ -0,0 +1,50 @@
+// This file was autogenerated by some hot garbage in the `nimbus-fml` crate.
+// Trust me, you don't want to mess with it!
+
+// This file was autogenerated by some hot garbage in the `nimbus-fml` crate.
+// Trust me, you don't want to mess with it!
+
+{%- for imported_module in self.imports() %}
+import {{ imported_module }}
+{%- endfor %}
+
+{% let nimbus_object = self.config.nimbus_object_name() -%}
+///
+/// An object for safely accessing feature configuration from Nimbus.
+///
+/// This is generated.
+public struct {{ nimbus_object }}Struct {
```

Comment:
Why add `Struct` here?

Reply:
Oh, I think I see. Do you think we should use statics here instead?

```swift
struct {{ nimbus_object }} {
    public static var api: FeaturesInterface? = nil
    public static let features = Features()
}
```

Reply:
Or, we follow the swift version of a singleton.

```swift
public struct {{ nimbus_object }} {
    public static let shared = { {{ nimbus_object }}() }()
}
```

I think I prefer this answer, but unsure if we use `shared`, `instance`, or `default` as the name.